demos/wascc/hello-world-assemblyscript/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "scripts": {
+        "graphql-lint": "graphql-schema-linter",
+        "gql2as": "wapc-gql2as schema.gql > assembly/module.ts && prettier --write assembly/module.ts",
+        "codegen": "npm run graphql-lint && npm run gql2as",
```

Comment:
Running this fails:

```
npm run codegen                                  

> @ codegen /Users/radu/projects/src/github.com/deislabs/krustlet/demos/wascc/hello-world-assemblyscript
> npm run graphql-lint && npm run gql2as


> @ graphql-lint /Users/radu/projects/src/github.com/deislabs/krustlet/demos/wascc/hello-world-assemblyscript
> graphql-schema-linter



✔ 0 errors detected


> @ gql2as /Users/radu/projects/src/github.com/deislabs/krustlet/demos/wascc/hello-world-assemblyscript
> wapc-gql2as schema.gql > assembly/module.ts && prettier --write assembly/module.ts

sh: wapc-gql2as: command not found
```

Not exactly sure where one can get that utility from.

Reply:
So this took me a while to hunt down... it’s hiding under https://github.com/wapc/as-codegen/tree/feature/initial_implementation (note the non-master branch). You have to compile from source.

I’ll add that to the docs